### DISCUSSION TITLE
Opening a new repository to implement upgrade/downgrade rules

### DISCUSSION THREAD

**Discussion by suprith-hub** [json-schema-org/community#709] [General]
As a part of GSoC project mentioned in this - https://github.com/json-schema-org/community/issues/599 issue. 
I will be taking up the project under my mentor @jviotti. 
Currently alterschema project in [this](https://github.com/sourcemeta/alterschema) repository has rules along with the engine which runs it.
Requesting:
1. Suggestion on the name of the repository like: JSON-Schema-Transform-Rules
2. Description - "A language agnostic set of upgrade/downgrade transformation rules for the JSON Schema specifications"
3. Permission for me and @jviotti to access over that repository.
4. Then, suggestion over the documentation README file of the repo. Documentation will have few more things like how to implement the engine, DSL implementation etc..
TO write new rules (which will be language agnostic) into the organization's new repository so that those rules can be used in any language(similar to JSON Schema test suite).
- thank you❤️‍🔥

**Comment by gregsdennis** [comment]
I don't think you needed to open a new discussion for this.  Probably something in Slack would have sufficed.

**Reply by suprith-hub** [reply]
So, I will take this into implementors channel right?

**Reply by gregsdennis** [reply]
I expect there will likely be a GSoC channel for this project.  Since @jviotti is the mentor for it, please let him handle those logistics.

**Reply by suprith-hub** [reply]
Cool!! actually he informed me that I can do this, thats good. I'll close the discussion.
cc @jviotti

**Reply by jviotti** [reply]
Will do!
